Mage/src/main/java/mage/game/events/SacrificedPermanentBatchEvent.java

```diff
@@ -0,0 +1,8 @@
+package mage.game.events;
+
+public class SacrificedPermanentBatchEvent extends BatchEvent<GameEvent> {
```

Comment:
The other existing batch events all enforce usage of a specific subclass to guard against bugs from putting the wrong type of event in a batch.